bundles/org.eclipse.osgi/supplement/src/org/eclipse/osgi/util/NLS.java

```diff
@@ -138,6 +139,16 @@ public static String bind(String message, Object[] bindings) {
 	 *   org/eclipse/example/nls/messages_en.properties
 	 *   org/eclipse/example/nls/messages.properties
 	 * </pre>
+	 * <p>
+	 * The properties files are read using the default encoding for resource bundles:
+	 * <ul>
+	 *   <li>In Java 8, resource files are read using ISO-8859-1 character encoding. Characters outside
+	 *   its range must be represented using Unicode escape sequences (e.g., <code>&#92;XXXX</code>).</li>
```

Comment:
It's kind of a nit, but is anything other than `\uXXXX` supported in a *.properties file?   It seems slightly wrong to leave out the `u`.

Reply:
Ah, nice catch. I erased `u` by accident while escaping the backslash.

Reply:
Pushed the fix.